.gitignore

```diff
@@ -1,6 +1,7 @@
 *.patch
 *.o
 *.swp
+.idea/
```

Comment:
please remove the `.gitignore` update from this commit. others looks good

Reply:
ok